README.md

```diff
@@ -548,6 +548,7 @@ For Kubernetes variants in AWS, the following settings are set for you automatic
 
 * `settings.kubernetes.cluster-dns-ip`: Derived from the EKS Service IP CIDR or the CIDR block of the primary network interface.
 * `settings.kubernetes.max-pods`: The maximum number of pods that can be scheduled on this node (limited by number of available IPv4 addresses)
+* `settings.kubernetes.hostname-override`: The node name kubelet uses as identification instead of the hostname or the name determined by the in-tree cloud provider if that's enabled. **Note:** This is currently only automatically generated in `aws-k8s-1.26` variants and set to the private DNS name of the EC2 instance and left unset for all other K8s variants.
```

Comment:
> I think it would be helpful to add a little information about if/how this overlaps with settings.network.hostname, as well as scenarios when someone may want/need to use this setting.

> about if/how this overlaps with settings.network.hostname

It does not overlap, and adding that might just be more confusing. 

I think the existing prose is sufficient. It really is just the node name kubelet uses to register itself. I'm wary about adding too much to these passthrough settings to kubelet since users can find the documentation for what it exactly does in kubelet.

Reply:
I agree with Zac. On a quick glance, `settings.network.hostname` and `settings.kubernetes.hostname-override` may seem related (if not directly correlated). A note on how they relate (or don't relate) would be ideal.